pkg/netmap/netmap.go

```diff
@@ -0,0 +1,61 @@
+package netmap
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
+)
+
+// Netmap represents netmap which contains preprocessed nodes.
+type Netmap struct {
+	Nodes Nodes
+}
+
+// NewNetmap constructs netmap from the list of raw nodes.
+func NewNetmap(infos []netmap.NodeInfo) (*Netmap, error) {
```

Comment:
What if there are `v2/netmap` and `v3/netmap` packages with `NodeInfo` structure. They can be equal, but still it is two different types for two major API versions. To support multiple version in netmap, consider using structures from `pkg/netmap`. 

This is a quite rough example: 

```go
package netmap // pkg/netmap

import v2netmap "github.com/nspcc-dev/neofs-api-go/v2/netmap"'
import v3netmap "github.com/nspcc-dev/neofs-api-go/v3/netmap"

type Node struct { 
  v2       *v2netmap.NodeInfo
  v3       *v3netmap.NodeInfo
  ID       uint64
  Index    int
  Capacity uint64
  Price    uint64
  AttrMap  map[string]string
}

func (n *NodeInfo) FromV2(info *v2netmap.NodeInfo) { ... }

func (n *NodeInfo) FromV3(info *v3netmap.NodeInfo) { ... }


func NewNetmap(infos []Node) (*Netmap, error) { ... }
```

Reply:
Renamed `NewNetmap` and `Node.NodeInfo` field to be more explicit about version.